### PR TITLE
Correct error message for student login to email/password should be a string

### PR DESCRIPTION
**Pull request by alexoragz**
## Description
The error being thrown incorrectly stated the email and password should be numbers. It should ask for strings.

## Motivation and Context
Stumbled on it while looking through the code.

## How Has This Been Tested?
Added a logging statement and saw the error message being logged to the console.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ X] I have read the **CONTRIBUTING** document.

### CODE REVIEW

File: src/ignitus-StudentLogin/actions.js [hunk 1]
```diff
@@ -7,14 +7,14 @@ import _ from 'lodash';
 
 export const logInRequest = (email, password) => {
   if (!_.isString(email))
-    throw new Error(`email must be number: ` + email);
+    throw new Error(`email must be a string: ` + email);
   if (!_.isString(password))
-    throw new Error(`password must be number: ` + password);
+    throw new Error(`password must be a string: ` + password);
```
Comment (outsourcegb) [inline src/ignitus-StudentLogin/actions.js]: Lets just not check about string password. We will have to allow strict password.

Comment (alexoragz) [reply src/ignitus-StudentLogin/actions.js]: Do you know what the requirements are? Like length and complexity.

Comment (outsourcegb) [reply src/ignitus-StudentLogin/actions.js]: For login, I would suggest to ignore other restrictions. Just shouldn't be blank.

Comment (outsourcegb) [reply src/ignitus-StudentLogin/actions.js]: Agree with you. 
Could you just remove them and add 
1# email should be valid email
2# password should not be empty

Also, remove this checking of form filling. It should check individually and prompt.

**Pull request closed without merging**